config/kubemark-config-infra.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: infrastructures.config.openshift.io
+spec:
+  group: config.openshift.io
+  names:
+    kind: Infrastructure
+    listKind: InfrastructureList
+    plural: infrastructures
+    singular: infrastructure
+  scope: Cluster
+  versions:
+  - name: v1
+    served: true
+storage: true
+---
+apiVersion: config.openshift.io/v1
+kind: Infrastructure
+metadata:
+  name: cluster
+status:
+  platform: AWS
```

Comment:
this need to be 'kubemark' cc @ingvagabund

Reply:
Right, if not set properly our Jenkins CI will start to fail.

Reply:
```
kubectl apply -f config/kubemark-config-infra.yaml
error: error validating "config/kubemark-config-infra.yaml": error validating data: [ValidationError(CustomResourceDefinition.spec.versions[0]): missing required field "storage" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1beta1.CustomResourceDefinitionVersion, ValidationError(CustomResourceDefinition): unknown field "storage" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1beta1.CustomResourceDefinition]; if you choose to ignore these errors, turn validation off with --validate=false
```